examples/connext_dds/waitset_status_cond/py/waitset_status_cond_subscriber.py

```diff
@@ -0,0 +1,92 @@
+# (c) Copyright, Real-Time Innovations, 2024.  All rights reserved.
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the software solely for use with RTI Connext DDS. Licensee may
+# redistribute copies of the software provided that all such copies are subject
+# to this license. The software is provided "as is", with no warranty of any
+# type, including any warranty for fitness for any purpose. RTI is under no
+# obligation to maintain or support the software. RTI shall not be liable for
+# any incidental or consequential damages arising out of the use or inability
+# to use the software.
+
+import argparse
+
+import rti.connextdds as dds
+
+from foo import Foo
+
+
+def subscriber_main(domain_id, sample_count):
+    # Create a DomainParticipant with default QoS
+    participant = dds.DomainParticipant(domain_id)
+
+    # Create a Topic and automatically register the type
+    topic = dds.Topic(participant, "Example waitset_status_cond", Foo)
```

Comment:
Use the same topic name as the other examples. May sure they interoperate.

```suggestion
    topic = dds.Topic(participant, "Example Foo", Foo)
```